app/Http/Controllers/PagesController.php

```diff
@@ -55,10 +55,17 @@ public function store()
 
         $page = new Page();
         if (Input::hasFile('hero_image')) {
-            $file = Input::file('hero_image');
-            $filename = $file->getClientOriginalName();
-            $file->move(public_path().'/pages/images/', $filename);
-            $page->hero_image = '/pages/images'.$filename;
+            $file = file_get_contents(Input::file('hero_image')->getRealPath());
+
+            // Set title in the database
+            $filename = Input::file('hero_image')->getClientOriginalName();
+            $page->hero_image = '/pages/images/'.$filename;
+
+            // Save file to S3 (or local storage)
+            $extension = Input::file('hero_image')->guessExtension();
+            $storagePath = uploadedContentPath('pages/images').'/'.$filename.'.'.$extension;
```

Comment:
This can be simplified a wee bit, I think. Here's how we [handle other uploads to S3](https://github.com/DoSomething/longshot/blob/6e0227abee60465cb0402d259863a1a2ad8d538d/app/Http/Controllers/ApplicationController.php#L136-L139).

Reply:
ahh @DFurnes I was pulling from [this function](https://github.com/DoSomething/longshot/blob/dd19dae0359c9317eb015059b1f6a9db9ded59f2/app/Scholarship/Repositories/SettingRepository.php#L84-L94)

Reply:
@DFurnes do we not need to get the extension there? That looks like the only bit here that could potentially be removed

Reply:
I _think_ it should handle the extension fine without that, since we don't use it for the uploads to the application. It'd be nice to consolidate all these to use that shared helper you linked so we don't have slightly different implementations in each place.

Reply:
Regardless, definitely not something that blocks merging this to get things back on track!

Reply:
@DFurnes sounds good! I'm going to merge and make a card for pulling all that logic into an image upload helper 👍